### PR TITLE
UI issue in Home screen

### PR DESCRIPTION
**Pull request by ShivanshGairola**
Targets #2536

### Description
the highlight is not the same size as the other two buttons.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/js/components/Dashboard/UserDashboardLibrary.vue [hunk 1]
```diff
@@ -22,9 +22,11 @@
         <li class="nav-item">
           <a
             id="recommend"
-            class="nav-link c-pointer"
+            class="nav-link c-pointer"style="
+            padding-bottom: 20px;
+            padding-top: 20px;"
```
Comment (gautam071) [inline resources/js/components/Dashboard/UserDashboardLibrary.vue:27]: Can we use bootstrap class?

**Review by gautam071 — DISMISSED** [review]
what is the purpose of removing the development.md file

Can you add it back?

File: resources/js/components/Dashboard/UserDashboardLibrary.vue (hunk 1 above)
Comment (ShivanshGairola) [reply resources/js/components/Dashboard/UserDashboardLibrary.vue]: done , I use py-3 for both top and bottom padding .

File: resources/js/components/Dashboard/UserDashboardLibrary.vue [hunk 2]
```diff
@@ -22,9 +22,8 @@
         <li class="nav-item">
           <a
             id="recommend"
-            class="nav-link c-pointer"
-            @click="setActiveTile('recommend')"
-            >Recommend</a
+            class="nav-link c-pointer py-3"
```
Comment (GauravGusain98) [inline resources/js/components/Dashboard/UserDashboardLibrary.vue:25]: Why you removed this `@click="setActiveTile('recommend')"`

It was calling the function `setActiveTile` when the element is clicked. Do we not need that functionality?

Comment (ShivanshGairola) [reply resources/js/components/Dashboard/UserDashboardLibrary.vue:25]: Changes have been reverted

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
The `recommend` button doesnot seem to have fix
image

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Comment by GauravGusain98** [comment]
> The `recommend` button doesnot seem to have fix image

This is fixed by running `npm run dev`

**Pull request merged**